### ISSUE TITLE
Data permission

### ISSUE THREAD

**Issue by GetSlained** [Rosewood-Development/PlayerParticles#118]
Hi, i want to make it so that players can only select a specific color for the dust_color_transition effect but there is no such permission for it.

is there a way i can do it?

**Comment by Esophose** [comment]
You're correct that there's no permission for it. There isn't any way to do that right now, sorry.

**Comment by GetSlained** [comment]
Is it going to be added in the next version?
